.gitignore

```diff
@@ -32,3 +32,4 @@ phpunit.xml
 
 # sqlite test database
 phinx_testing.sqlite3
+/nbproject/
```

Comment:
These kinds of user-specific ignores should be done via [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) not at the repository level.